ksqldb-common/src/main/java/io/confluent/ksql/schema/ksql/SystemColumns.java

```diff
@@ -129,12 +129,17 @@ public static boolean isSystemColumn(final ColumnName columnName) {
     private static final Set<ColumnName> VERSION_ONE_NAMES;
 
     static {
-      VERSION_ZERO_NAMES = new HashSet<>();
-      VERSION_ZERO_NAMES.add(ROWTIME_NAME);
-
-      VERSION_ONE_NAMES = new HashSet<>(VERSION_ZERO_NAMES);
-      VERSION_ONE_NAMES.add(ROWOFFSET_NAME);
-      VERSION_ONE_NAMES.add(ROWPARTITION_NAME);
+      ImmutableSet.Builder<ColumnName> versionZeroBuilder = ImmutableSet.builder();
+      VERSION_ZERO_NAMES = versionZeroBuilder
+          .add(ROWTIME_NAME)
+          .build();;
+
+      ImmutableSet.Builder<ColumnName> versionOneBuilder = ImmutableSet.builder();
+      VERSION_ONE_NAMES = versionOneBuilder
+          .addAll(VERSION_ZERO_NAMES)
+          .add(ROWPARTITION_NAME)
+          .add(ROWOFFSET_NAME)
+          .build();
```

Comment:
I like this way of making the sets immutable vs making a `HashSet` then using the `ImmutableSet(Iterable)` constructor. This is scalable as we can always just create a versionTwoBuilder then add all from versionOne. Let me know if this works though